scripts/network.js

```diff
@@ -282,6 +279,11 @@ export class ExplorerNetwork extends Network {
                 ? 0
                 : nBlockHeights.sort((a, b) => a - b).at(-1);
         this.fullSynced = true;
+        createAlert('success', translation.syncStatusFinished, 12500);
+        getEventEmitter().emit('sync-status-update', 0, 0, true);
+    }
+    reset() {
+        this.fullSynced = false;
```

Comment:
shouldn't we reset also `blocks` and `lastBlockSynced`?

Reply:
I think they get reset by `walletFullySync` later on, but it's probably a good idea to reset everything